mernarezk/assignment1/Q2.java

```diff
@@ -0,0 +1,37 @@
+package Assignment1;
+
+public class Q2 {
+	/**
+	 * Method that gets Kth node from back of a singly linkedlist.
+	 * @param head of the linkedlist
+	 * @param kth node to be returned.
+	 */
+	public ListNode getKth(ListNode head, int k) {
+		ListNode first = head;
+		for (int i = 0; i < k; i++) {
+			// if K > length of linkedlist.
+			if (first == null)
+				return null;

```

Comment:
If k is greater than the size of the list we should probably throw an exception instead of returning null.